<?xml version="1.0"?>
<launch>

  <node pkg="lidar_pointcloud_obstacle" type="LidarPointcloudObstacleNode" name="LidarPointcloudObstacle"  output="screen">
    <param name="elevation_resolution" value="0.01" />
    <param name="local_map_frame_id" value="odom" />
    <param name="resolution_xy" value="0.02" />
    <param name="max_grid_size_x" value="1200" />
    <param name="max_grid_size_y" value="1200" />
    <param name="unknown_elevation" value="50" />
  </node>
</launch>




